src/SFA.DAS.QnA.Database/Script.PostDeployment.sql

```diff
@@ -18,4 +18,14 @@ Post-Deployment Script Template
 :r .\OneOffScripts\APR-1583.sql
 :r .\OneOffScripts\APR-1650.sql
 :r .\OneOffScripts\APR-1779.sql
--- END
\ No newline at end of file
+-- END
+
+
+-- Finally rebuild indexes on all tables, helps keep fragmentation down
+ALTER INDEX ALL ON [Projects] REBUILD;
+ALTER INDEX ALL ON [Workflows] REBUILD;
+ALTER INDEX ALL ON [WorkflowSequences] REBUILD;
+ALTER INDEX ALL ON [WorkflowSections] REBUILD;
```

Comment:
I'm not sure you need rebuilds on projects and workflows, they really don't change much.... but not a problem if you leave this in

Reply:
I was on the fence when I put this in. Thing is that the import script messes with those tables so decided it was probably worth including